jobs/integr8ly/alerts-test.yaml

```diff
@@ -0,0 +1,70 @@
+---
+
+- job:
+    name: alerts-test
+    description: 'Executes alert test used to check if all of the alerts except DeadMansSwitch are green and if ApicuritoPodCount alerts gets triggered as expected'
+    project-type: pipeline
+    sandbox: true
+    parameters:
+      - string:
+          name: REPOSITORY
+          default: 'https://gitlab.cee.redhat.com/integreatly-qe/integreatly-qe.git'
+          description: 'QE repository containing the smoke tests source code.'
+      - string:
+          name: BRANCH
+          default: 'master'
+          description: 'Branch of the repository'
+      - string:
+          name: CLUSTER_URL
+          description: 'URL of cluster on which the smoke tests will be executed.'
```

Comment:
Super minor - but the description should be updated. Same as above for the repository

Reply:
good catch. Thanks! changed the description